docs/how-to/platform-console.md

```diff
@@ -102,21 +60,20 @@ To create a private plugin:
 
 ## Platform Changes Sidebar
 
-The Platform Console introduces a new **_Platform_ Changes** sidebar that replaces the **_Pending_ Changes** sidebar. It's only visible when you're viewing the Platform Console.
+The Platform Console introduces a new **Platform Changes** sidebar that shows changes made to platform-wide settings and configurations. This is separate from the **Pending Changes** sidebar, which shows service-level changes.
```

Comment:
Actually the private plugins and templates are not "platform-wide" but they are per project (- we are adding the option now to use private plugins and templates of a specific project in all the projects if defined so.
Your call whether we need to update it or not